pkgs/by-name/ra/rainbowcrack/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitLab
+, alglib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8-0kali1";
```

Comment:
version should be 1.8 to be consistent with 
https://repology.org/project/rainbowcrack/versions